src/modules/forms/plugins/Compression/CompressionPlugin.js

```diff
@@ -25,7 +26,10 @@ const CompressionPlugin = ({ className, name, handlePluginExclude, pluginName })
           <Label>Plugin Name</Label>
           <Input input={{value: 'Compression'}} disabled />
         </Row>
-        <ControlBar name={`${name}.enabled`} removePlugin={() => handlePluginExclude(pluginName)} />
+        {
+          !disabled &&
```

Comment:
Can we somehow check if `disabled` is true in the `Control` and depending on that return null or the component because we seem to be doing that a number of times.  You can also check out `branch` from `recompose`: https://github.com/acdlite/recompose/blob/master/docs/API.md#branch.

Reply:
There is a sense in your suggestion, @ngenerio, and I would use it as I've done with some other components. 
But in this particular case, I think it's more correct to use this approach. I definitely need to change property name from `disabled` to `isPreview` or something like that... because with this check we just detect if it is a regular form (then we should render as before) or just a preview page (in that case we don't need Controls at all, which means the `Control` component even should not know anything about what's going on on the higher level).

What I'm trying to say - `Control` doesn't depend on this `disable` property, that's why it's not his responsibility to decide to render or return `null`.